templates/agenda_case_item/documents.html.twig

```diff
@@ -45,7 +45,7 @@
                         <div class="btn-group" role="group" aria-label="{% trans %}List functions{% endtrans %}">
                             <!-- Delete button -->
                             {% if not agenda.isFinished() %}
-                            <form id="myform" class="" method="post" action="{{ path('agenda_case_item_document_delete', {'id': agenda.id, 'agenda_item_id': agenda_item.id, 'document_id': document.id}) }}" onsubmit="return confirm('{% trans %}Are you sure you want to delete this item?{% endtrans %}');">
+                            <form class="" method="post" action="{{ path('agenda_case_item_document_delete', {'id': agenda.id, 'agenda_item_id': agenda_item.id, 'document_id': document.id}) }}" onsubmit="return confirm('{% trans %}Are you sure you want to delete this item?{% endtrans %}');">
```

Comment:
See the comment on how to handle the `confirm` message above.

Reply:
Good catch. This is something that will be needed in many other places, so we will fix it in a different branch.